build/ci-build.yml

```diff
@@ -89,8 +89,6 @@ stages:
         steps:
           - task: FuncToolsInstaller@0
             displayName: 'Install Azure Functions Core tools'
-            inputs:
-              version: '4.0.5390'
```

Comment:
Removed this pinned version again, and works now with the latest CLI tool and .NET 8.